packages/esm-patient-orders-app/src/lab-results/lab-results-form.component.tsx

```diff
@@ -23,12 +30,21 @@ import { createLabResultsFormSchema } from './lab-results-schema.resource';
 
 import ResultFormField from './lab-results-form-field.component';
 import styles from './lab-results-form.scss';
+import { type ConfigObject } from '../config-schema';
 
 export interface LabResultsFormProps extends DefaultPatientWorkspaceProps {
-  order: Order;
+  order: Order & {
+    fulfillerStatus: 'DRAFT' | 'COMPLETED' | Order['fulfillerStatus'];
```

Comment:
The [public enum FulfillerStatus {](https://github.com/openmrs/openmrs-core/blob/main/api/src/main/java/org/openmrs/Order.java#L65-L66) doesn't support a `DRAFT` status. Would we need work to add support for it as a precursor to getting this merged in?